games-emulation/stella/stella-5.1.3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils
```

Comment:
where are you using `gnome2-utils`?

Reply:
He will do when he adds the icon cache stuff I requested.